pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -446,6 +446,8 @@ message GenerativeSplitAndScatterSpec {
   repeated jobs.jobspb.RestoreProgress.FrontierEntry checkpointed_spans = 21 [(gogoproto.nullable) = false];
   // MaxFileCount is the max number of files in an extending restore span entry.
   optional int64 max_file_count = 22 [(gogoproto.nullable) = false];
+  // SQLInstanceIDs is a slice of SQL instance IDs available for dist restore.
+  repeated int32 sql_instance_ids = 23[(gogoproto.nullable) = false, (gogoproto.customname) = "SQLInstanceIDs"];
```

Comment:
Hrm. It seems a little odd to me to change the field ID here. In the other PRs it is field 24. 

Because of how we plan the SplitAndScatter processors, I think we always know the gateway node and processor node is the same, so perhaps this isn't an issue.

But perhaps we should keep the IDs the same in each of these PRs just in case.

Reply:
updated the field id and comment as discussed offline